server/routes/charts.js

```diff
@@ -213,4 +213,30 @@ router
       return res.status(404).json({ message: 'Chart was not duplicated' })
     }
   })
+
+router
+  .route('/api/v1/charts/:chart_id/shared')
+  .patch(ensureAuthenticated, async (req, res) => {
+    try {
+      const { chart_id } = req.params
+      const { dataDb } = req.app.locals
+      const { sharedWith } = req.body
+      const { result } = await dataDb.collection(collections.charts).updateOne(
+        { _id: new ObjectID(chart_id) },
+        {
+          $set: {
+            sharedWith,
+            updatedAt: new Date().toISOString(),
+          },
+        }
+      )
+
+      return result.ok === 1
+        ? res.status(200).json({ data: result })
+        : res.status(404).json({ message: 'Chart could not be shared' })
+    } catch (error) {
+      return res.status(404).json({ message: 'Chart was not shared' })
```

Comment:
why is this error message different from the error message above?

Reply:
this message is still different from the one above. In this function we're either returning `res.status(404).json({ message: 'Chart could not be shared' })` or `res.status(404).json({ message: 'Chart was not shared' })` for error states. Can we make those the same?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/131/commits/568515fe14fa486723bc590ae604e9a9794053a2